src/location-input/zipData/zipStore.ts

```diff
@@ -9,6 +9,7 @@ export const getZipStore = (sheetConfig: SheetDataConfig) => {
   const load = async () => {
     try {
       const res = await fetchZipCodes(sheetConfig);
+      console.log("response: ", res);
```

Comment:
I'm not sure ATM if the build removes console logs but i'd remove this

Reply:
thank you